app/controllers/tmp/iterations_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Tmp::IterationsController < ApplicationController
+  def create
```

Comment:
This action does the same thing as the `submit_iteration` script but is runnable via clicking a button instead. As neither will ever get near production, I don't mind the duplication for now.